components/tooltip/theme.css

```diff
@@ -0,0 +1,229 @@
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-utilities';
+
+:root {
+  --unit: 10px;
+
+  --tooltip-arrow-size: calc(.8 * var(--unit));
+  --tooltip-arrow-position-correction: calc(0.45 * var(--unit));
+  --tooltip-margin: calc(0.8 * var(--unit));
+  --tooltip-border-radius: calc(0.4 * var(--unit));
+  --tooltip-border-width: 1px;
+  --tooltip-box-shadow: 0 3px 12px 0 color(var(--color-teal-darkest) a(10%));
+  --tooltip-max-width: calc(18 * var(--unit));
+  --tooltip-small-padding-horizontal: calc(1.2 * var(--unit));
+  --tooltip-small-padding-vertical: calc(0.6 * var(--unit));
+  --tooltip-medium-padding-horizontal: calc(1.2 * var(--unit));
+  --tooltip-medium-padding-vertical: calc(1.2 * var(--unit));
+  --tooltip-animation-duration: 200ms;
+  --tooltip-z-index: 200;
+}
+
+.label,
+.tooltip {
+  svg {
+    color: inherit;
+    fill: currentColor;
+    -ms-transform: translateY(-1px);
+
+    * {
+      fill: currentColor;
+    }
+  }
+}
+
+.label {
+  background: color(var(--color-teal-darkest) a(8%));
+  border-radius: var(--tooltip-border-radius);
+  color: var(--color-teal-darkest);
+  padding: var(--spacer-smallest) var(--spacer-smaller);
+
+  svg {
+    margin-right: var(--spacer-smallest)
+  }
+
+  &.inverse {
+    background: var(--color-teal-darkest);
+    color: var(--color-teal-lightest);
+  }
+}
+
+.tooltip {
+  display: block;
+  padding: var(--tooltip-margin);
+  pointer-events: none;
+  position: absolute;
+  text-align: left;
+  transform-origin: top left;
+  transition: cubic-bezier(0.4, 0, 0.2, 1) var(--tooltip-animation-duration) transform;
+  z-index: var(--tooltip-z-index);
+
+  &::before {
```

Comment:
**PP:** I know this is the official way of writing pseudo selectors, but AFAIK we've always used single colons `:before` instead of double `::before`. My OCD is killing me, so either we change it everywhere to double or these to single 😬

Reply:
`:before` is the css2 way of writing the before pseudo element .
`::before` is the css3 way of writing the before pseudo element to distinguish pseudo classes (`:focus`, `:active`, ...) from pseudo elements. I would opt for the replace everywhere solution :)